classes/addons/polylang.php

```diff
@@ -104,14 +104,18 @@ public function bea_csf_set_term_language( array $emitter_data, $sync_fields, WP
 	/**
 	 * Add additional polylang data for post
 	 *
-	 * @param array $emitter_data
+	 * @param array|bool $emitter_data
 	 *
-	 * @return array
+	 * @return array|bool
 	 *
 	 * @author Léonard Phoumpakka
 	 *
 	 */
-	public function bea_csf_server_posttype_merge( array $emitter_data ) {
+	public function bea_csf_server_posttype_merge( $emitter_data ) {
```

Comment:
On peut pas mettre juste `array $emitter_data` afin de forcer array ?

Reply:
cela génère cette erreure lorsque le type array est précisé

`PHP Fatal error:  Uncaught TypeError: Argument 1 passed to BEA_CSF_Addon_Polylang::bea_csf_server_posttype_merge() must be of the type array, bool given, called in /var/www/cib/root/web/wp/wp-includes/class-wp-hook.php on line 309 and defined in /var/www/cib/root/web/app/plugins/bea-content-sync-fusion/classes/addons/polylang.php:116`

quand on remonte dans le filtre qui défini emitter_data il semble qu'il puisse renvoyer `false `

Reply:
Ok, à ne pas mettre alors